cli/src/commands/content/createChannel.ts

```diff
@@ -0,0 +1,78 @@
+import ContentDirectoryCommandBase from '../../base/ContentDirectoryCommandBase'
+import { IOFlags, getInputJson, saveOutputJson } from '../../helpers/InputOutput'
+import { NewAsset} from '@joystream/types/content'
+import {ChannelMetadata} from '@joystream/content-metadata-protobuf'
+import { Vec, Option} from '@polkadot/types';
+import AccountId from '@polkadot/types/generic/AccountId';
+import { Bytes } from '@polkadot/types/primitive';
+
+type ChannelCreationParametersInput = {
+  assets: Vec<NewAsset>,
+  meta: ChannelMetadata.AsObject,
+  reward_account: Option<AccountId>,
+}
+
+type ChannelCreationParameters = {
+  assets: Vec<NewAsset>,
+  meta: Bytes,
+  reward_account: Option<AccountId>,
+}
+
+export default class CreateChannelCommand extends ContentDirectoryCommandBase {
+  static description = 'Create channel inside content directory.'
+  static flags = {
+    context: ContentDirectoryCommandBase.contextFlag,
+    ...IOFlags,
+  }
+
+  async run() {
+    let { context, input, output } = this.parse(CreateChannelCommand).flags
+
+    if (!context) {
+      context = await this.promptForContext()
+    }
+
+    const currentAccount = await this.getRequiredSelectedAccount()
+    await this.requestAccountDecoding(currentAccount)
+
+    const actor = await this.getActor(context)
+
+    if (input) {
+      let channelCreationParametersInput = await getInputJson<ChannelCreationParametersInput>(input)
```

Comment:
Good point, but it would be much harder to accomplish the same with Videos and protobuf seems to be fairly flexible, so there would probably be lots of adjustments in the future. Let`s go as is now if you don`t mind.